### DISCUSSION TITLE
Network error: Response not successful: Received status code 413

### DISCUSSION THREAD

**Discussion by Ivanova26** [requarks/wiki#7356] [Error / Bug Report] [answered]
I get the above error when trying to save a large page. The page size is large due to 2 draw.io.

I have upgrade the Admin/Security/MaxUploadSize and edit this line to 100mb but i still have the same error. Even after restarting the container.
https://github.com/Requarks/wiki/blob/a1f1cd7356ddc2ba9054aa0f39b0b9f3f43ce3f8/server/master.js#L94

When I look into the development tools of my browser (I've tried multiple browser but still have the same issue) i got this error about graphql: "413 Payload Too Large"

Note that I'm using Caddy as reverse proxy and it's hosted on another VM.

Host info:
OS: Docker
Wiki.js version: 2.5.303

**Comment by Ivanova26** [comment, marked as answer]
After a few more hours testing things, I've found a solution.
You need to add `bodyParserLimit: 100mb` at your config.yml inside the container (with docker run).

**Comment by ulysserevient** [comment]
Thx
It solved my problem too

**Comment by chchia** [comment]
thanks, this solved my problem too, wondering when will wiki.js have this setup available in admin config page.
